packages/compass-global-writes/src/components/index.spec.tsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import { expect } from 'chai';
-import { render, screen } from '@mongodb-js/testing-library-compass';
+import { screen } from '@mongodb-js/testing-library-compass';
 import { GlobalWrites } from './index';
+import { renderWithStore } from './../../tests/create-store';
 
 describe('Compass GlobalWrites Plugin', function () {
-  it('renders a Plugin', function () {
-    render(<GlobalWrites />);
-    expect(screen.getByText('This feature is currently in development.')).to
-      .exist;
+  it('renders plugin in NOT_READY state', function () {
+    renderWithStore(<GlobalWrites shardingStatus={'NOT_READY'} />);
+    expect(screen.getByText(/loading/i)).to.exist;
+  });
+
+  it('renders plugin in UNSHARDED state', function () {
+    renderWithStore(<GlobalWrites shardingStatus={'UNSHARDED'} />);
+    expect(screen.getByTestId('shard-collection-button')).to.exist;
+  });
```

Comment:
can we add SUBMITTING_FOR_SHARDING?

Reply:
added in [59787bb](https://github.com/mongodb-js/compass/pull/6289/commits/59787bb7983cd9b6b8a72c7bf8a010b88824d66d)